services/fluxmonitor.go

```diff
@@ -40,32 +40,32 @@ type FluxMonitorConfig struct {
 func ParseFMSpec(jsonSpec json.RawMessage, runtimeConfig store.RuntimeConfig) (FluxMonitorConfig, error) {
 	var fmConfig FluxMonitorConfig
 
-	res := gjson.GetBytes(jsonSpec, "feeds.#.url")
+	res := gjson.GetBytes(jsonSpec, "fluxmonitor.feeds.#.url")
```

Comment:
Maybe keep only the fluxmonitor part before starting parsing? In order to not duplicate the `fluxmonitor.` below

Reply:
I initially wanted to do that, but wasn't able to get what I was looking for. Do you know how to do this with gjson?

Reply:
Something like 

```jsonSpec = []byte(gjson.GetBytes(jsonSpec, "fluxmonitor").Raw)```

or rename it to `fmSpec`